environment.py

```diff
@@ -350,4 +350,13 @@ def env() -> Mapping[str, Optional[str]]:
 
         # This is set only during integration tests. Never override this to 1.
         'AZUL_TEST_MODE': '0',
+
+        # ARN-style configuration for The Google Cloud project and BigQuery
+        # dataset name for the dataset/snapshot to be indexed from the
+        # Terra Data Repository.
+        # Syntax is gbq:{project-name}:{"dataset"|"snapshot"}/{dataset-name}
+        # Note that if the target is a TDR "dataset" (as opposed to a snaphot),
+        # the name of the coresponding BigQuery dataset (used here) will not
+        # match the name used by Terra.
```

Comment:
This will force us to rely on the undocumented convention for TDR to prepend `datarepo_` to the names of the BigQuery datasets corresponding to TDR datasets.